src/app/Providers/components/ProvidersTable/VMware/VMwareProvidersTable.tsx

```diff
@@ -50,11 +51,11 @@ const VMwareProvidersTable: React.FunctionComponent<IVMwareProvidersTableProps>
 
   const { selectedItems, toggleItemSelected, areAllSelected, selectAll } = useSelectionState<
     IVMwareProvider
-  >(sortedItems);
+  >({ items: sortedItems });
   const {
     selectedItems: expandedProviders,
     toggleItemSelected: toggleProviderExpanded,
-  } = useSelectionState<IVMwareProvider>(sortedItems);
+  } = useSelectionState<IVMwareProvider>({ items: sortedItems });
```

Comment:
Let's use `isEqual` and `isItemSelected` here as well. Providers already have `metadata.uid` in them in our mocks... oh, but they duplicate the same uid in the mock files. when we make the mock providers with the different names in the mocks let's also give them unique uids. Unless you want to use names as the identifier?

Reply:
Good call - I am just going to use name for now & open an issue to circle back on generating UIDs for providers. Also I think we may need to consider the same for mappings.

Reply:
Sounds good!